CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Release History
-## 1.0.0 (2012-11-01)
+
+## 1.1.0 (2023-01-19)
+
+### Behavior Changes
+- `SnowparkSourceCodeExtractionException` would be raised when Snowpark fails to extract the source code of a function referenced by UDF registration.
+- `Session.create_dataframe()` no longer ignores the optional parameter `schema` when `data` is a pandas DataFrame.
+
+### New Features:
+- Added `asc`, `asc_nulls_first`, `asc_nulls_last`, `desc`, `desc_nulls_first`, `desc_nulls_last`, `date_part` in functions.
+- Enabled creating UDFs that reference non-local functions. This feature is not available inside stored procedures.
+- Added the property `DataFrame.dtypes` to return a list of column name, data type pairs.
+- Added the following aliases:
+  - `functions.expr()` for `functions.sql_expr()`.
+  - `functions.date_format()` for `functions.to_date()`.
+  - `functions.monotonically_increasing_id()` for `functions.seq8()`
+  - `functions.from_unixtime()` for `functions.to_timestamp()`
+
+### Bug Fixes:
+- Fixed a bug in SQL simplifier that didn’t handle Column alias and join well in some cases.
+- Fixed a bug where `Session.create_dataframe` ignored user-specified `schema` and used inferred schema when the input data is a pandas DataFrame.
+
+### Improvements
+- Session parameter `PYTHON_SNOWPARK_USE_SQL_SIMPLIFIER` will be True when Snowflake 7.3 is released. In snowpark-python, `session.sql_simplifier_enabled` reads the value of `PYTHON_SNOWPARK_USE_SQL_SIMPLIFIER` by default, meaining SQL simplfier is enabled by default after Snowflake 7.3 release. To turn this off, set `PYTHON_SNOWPARK_USE_SQL_SIMPLIFIER` in Snowflake to False or run `session.sql_simplifier_enabled = False` from snowpark. You’re recommended to use the SQL simplifier to generate more concise SQLs unless you hit a bug.
```

Comment:
Shall we rephrase `You’re recommended to use the SQL simplifier to generate more concise SQLs unless you hit a bug`?

Reply:
Yeah especially if @sfc-gh-mabrennan have any good suggestions.

Reply:
How about something like this:

```
The session parameter `PYTHON_SNOWPARK_USE_SQL_SIMPLIFIER` will be `True` after Snowflake 7.3 is released. In snowpark-python, `session.sql_simplifier_enabled` reads the value of `PYTHON_SNOWPARK_USE_SQL_SIMPLIFIER` by default, meaning that the SQL simplfier is enabled by default after the Snowflake 7.3 release. To turn this off, set `PYTHON_SNOWPARK_USE_SQL_SIMPLIFIER` in Snowflake to `False` or run `session.sql_simplifier_enabled = False` from Snowpark. It is recommended to use the SQL simplifier because it helps to generate more concise SQL.
```

Reply:
I'm not sure what the best thing to say about the "unless you hit a bug" part of it. Maybe something like:

"You might want to disable the SQL simplifier in some situations if a bug occurs."

Optionally, it would be better to be a bit more specific about the why part, if that's possible.

Reply:
Should we mention "unless you hit a bug"? It makes me feel simplifier is buggy 😅

Reply:
I agree with Mary, I think we should either (1) just leave it out or (2) mention specific unsupported usage.
cc: @sfc-gh-yixie

Reply:
I agree to remove the "bug" mention.